src/modules/UI/scenes/CreateWallet/CreateWalletSelectFiat.ui.js

```diff
@@ -0,0 +1,191 @@
+// @flow
+
+import React, {Component} from 'react'
+import {Actions} from 'react-native-router-flux'
+import {
+  ActivityIndicator,
+  Alert,
+  View,
+  Keyboard,
+  TouchableHighlight,
+  Image
+} from 'react-native'
+import Text from '../../components/FormattedText'
+import {SecondaryButton, PrimaryButton} from '../../components/Buttons'
+import {FormField} from '../../../../components/FormField.js'
+import SearchResults from '../../components/SearchResults'
+
+import styles, {styles as stylesRaw} from './style.js'
+import s from '../../../../locales/strings.js'
+import PLATFORM from '../../../../theme/variables/platform'
+import Gradient from '../../components/Gradient/Gradient.ui.js'
+import * as UTILS from '../../../utils'
+import type
+{
+  GuiWalletType,
+  GuiFiatType,
+  FlatListItem,
+  DeviceDimensions
+} from '../../../../types'
+
+const WALLET_NAME_INPUT_PLACEHOLDER = s.strings.fragment_wallets_addwallet_name_hint
```

Comment:
I know this was initially done in some files, but do not add an extra level of indirection for strings. Just use the s.strings.XXX variable wherever needed. The extra const doesn't help in any way and just adds more work